packages/onboarding/src/onboarding-feature-generate.tsx

```diff
@@ -63,8 +66,12 @@ export function OnboardingFeatureGenerate({ redirectTo }: { redirectTo: string }
             <div>
               <OnboardingUiMnemonicSelectStrength setStrength={setStrength} strength={strength} />
             </div>
+            <Button onClick={() => setRevealed((value) => !value)} type="button" variant="secondary">
+              <UiIcon icon="watch" />
+              {revealed ? 'Hide mnemonic' : 'Show mnemonic'}
```

Comment:
We probably want to translate these.

Reply:
Updated. Good catch. I did not think of this because in theory you can't currently switch language until after onboarding.